libs/designer-ui/src/lib/picker/filepickerPopover.tsx

```diff
@@ -0,0 +1,68 @@
+import { FilePickerPopoverHeader } from './filepickerPopoverHeader';
```

Comment:
Looks like there's an a11y issue where you can't tab from header to the file rows.

Reply:
Fixed.